### DISCUSSION TITLE
LinkChecker-GUI v10.0.0a2

### DISCUSSION THREAD

**Discussion by cjmayo** [linkchecker/linkchecker#707] [Announcements]
Get "File/Save Results..." working on PyQt5.

**Comment by jayavenkats** [comment]
i followed the instructions for installation of linkchecker-gui from command prompt and installed it using pip3 install.
in the usage section it is mentioned to use $ linkchecker-gui but getting $ is not recognized as an internal or external command. 
can you help me how to create the gui entry in desktop

**Reply by cjmayo** [reply]
Good point, the use of `$ ` isn't consitent with the rest of the document. The command is just `linkchecker-gui`.

Getting an entry is an OS specific question.

The pip package installs a freedesktop.org specification compliant entry that will work on Linux and others that follow the standard.

I don't expect it to work if installing directly into MS Windows. Installing inside a recent version of Windows Subsystem for Linux (WSL) might give an entry though.  https://learn.microsoft.com/en-us/windows/python/web-frameworks
